### ISSUE TITLE
Embedded artwork: do not show not-square images

### ISSUE THREAD

**Issue by leandroalonso** [Automattic/pocket-casts-ios#1170]
When displaying RSS artwork, some of them will have a rectangular aspect. Eg.:

| Now Playing artwork | Now Playing |
| -------------------- | ------------- |
| ![IMG_2384](https://github.com/Automattic/pocket-casts-ios/assets/7040243/2ed8ecb0-e57b-43f3-a35b-aecdfa344486) | ![IMG_2388](https://github.com/Automattic/pocket-casts-ios/assets/7040243/7a2ae152-dc76-4ea5-9391-21e67726526a) |

For this kind of content, we should ignore it and not show it.

**Comment by BKrnk1337** [comment]
I discovered something similar. But in my case is more the aspect ratio.


https://github.com/Automattic/pocket-casts-ios/issues/1084

**Comment by erricgunawan** [comment]
A feedback from 7352711-zen:

> I think if the podcast creator embeds a non-square artwork into the episode it should still be honored by the podcast player. Rectangular art in the Now Playing lockscreen and notification in iOS is correctly handled and displayed, and there shouldn't be a need to hide it from the listener.

> I just wanted to send feedback on a proposed change that would negatively affect my own personal experience using the app.

**Comment by rviljoen** [comment]
Agreed - I think this one should be closed. In contrast, #1084 should be fixed
